packages/starlight/virtual.d.ts

```diff
@@ -7,3 +7,16 @@ declare module 'virtual:starlight/plugin-translations' {
 	const PluginTranslations: import('./utils/plugins').PluginTranslations;
 	export default PluginTranslations;
 }
+
+declare module 'virtual:starlight/project-context' {
```

Comment:
Maybe we even add a note to that effect?

```suggestion
// TODO: Move back to `virtual-internal.d.ts` when possible. For example, when dropping support for
// legacy collections, `utils/translations.ts` would no longer need to import project context.
declare module 'virtual:starlight/project-context' {
```

Reply:
Good call, I guess this would also highly change when we get back to bundling but it's an easy comment making sure we don't forget 👍